README.md

```diff
@@ -33,30 +33,49 @@ Quoting from [The Tao of IETF](https://www.ietf.org/about/participate/tao/), **e
+See the documents in this repo titled [LICENSE.md](https://github.com/WebOfTrust/Keri/blob/main/LICENSE.md) and [CONTRIBUTING.md](https://github.com/WebOfTrust/Keri/blob/main/CONTRIBUTING.md) for licensing and contributing policy. Besides the IETF licensing terms, these include the comprehensive Apache2 license for all associated intellectual property (IP) including patents. The advantage of one comprehensive license for all contributions is that there will always be alignment between all contributors and for any type of contribution. This alignment includes an **inbound=outbound** [policy](https://opensource.guide/legal/) for all related IP. We only want contributions to KERI made here in the WebOfTrust project that are licensed as free, and [nonreciprocal](https://opensource.org/node/875) open source be it software or specification.  
 
 ## Meetings
 
-We meet every other Tuesday at 10 am EDT (or EST). (First meeting was  2021 October, 19 )
-The zoom link is here:
-https://us02web.zoom.us/j/89842900185?pwd=OEpXSU5pa29FR1A3OXFkUithZFNwQT09
-Meeting ID: 898 4290 0185
-Passcode: 300255
+### Dev call
+We meet every Thursday at 10 a.m. EDT (or EST).
+The Zoom link is here:
+https://us06web.zoom.us/j/81679782107?pwd=cTFxbEtKQVVXSzNGTjNiUG9xVWdSdz09
+Meeting ID: 816 7978 2107
+Passcode: 379242
+
+### Specification call
+We meet every Tuesday at 10  a.m EDT (or EST) as part of the ACDC/KERI Task Force under the Technology Stack Working Group at the Trust over IP Foundation:
+
+Please download and import the following iCalendar (.ics) files to your calendar system.
+Weekly: https://us06web.zoom.us/meeting/tZUrcOCvpjkrGtMfRtTcLyp3nxnR2_7XMbgW/ics?icsToken=98tyKuGpqDwjHdyXsRmARpwcBI_CXe3xiHpcjbdwmi3xNyN_exOlOPpMAOBAR-D9
+
+Join Zoom Meeting
+https://us06web.zoom.us/j/81679782107?pwd=cTFxbEtKQVVXSzNGTjNiUG9xVWdSdz09
 
-The meeting agenda may be found [here](https://github.com/WebOfTrust/keri/blob/main/agenda.md)
+Meeting ID: 816 7978 2107
+Passcode: 379242
+
+### Implementors call
+We meet every Thursday at 10 am EDT (or EST) under the Web Of Trust:
 
 ### Related Meetings
-A closely related specification that leverages KERI is ACDC. This is hosted at ToIP and meets on the alternate weeks at the same time as the KERI meetings. These meetings complement the KERI meetings reference above. For information look here [ToIP ACDC Task Force](https://wiki.trustoverip.org/display/HOME/ACDC+Meeting+Page) and here [ACDC IETF Draft](https://github.com/trustoverip/tswg-acdc-specification).
+A closely related specification that leverages KERI is ACDC. This is hosted at ToIP and meets on alternate weeks at the same time as the KERI meetings. These meetings complement the KERI meetings referenced above. For information look here [ToIP ACDC Task Force](https://wiki.trustoverip.org/display/HOME/ACDC+Meeting+Page) and here [ACDC IETF Draft](https://github.com/trustoverip/tswg-acdc-specification).
```

Comment:
LGTM except for this section which I think can be deleted or merged with above sections.  

eg) 
Line 51 is `### Dev call`
Line 58 is `### Specification call`
Line 70 is `### Implementors call`
Line 73 is `### Related Meetings` 

so they're repeated twice I think.  Or you could commit this one can I could merge no problem.

Reply:
I agree that we should have the Dev Call and Specification call and the other two can get removed